<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WirelessAlarmWidget</class>
 <widget class="QWidget" name="WirelessAlarmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wireless Alarm System</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Alarm Phone Number:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="alarmPhoneEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Soft Keyboard</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="num6Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="num7Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="num8Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="num9Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="num2Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="num3Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="num4Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="num5Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="num1Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="num0Btn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="numPlusBtn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="backBtn">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>&lt;-</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Current phone:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="currentPhoneNumber">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>IR Sensor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Gas Sensor</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="TwinklingLabel" name="irSensorLabel">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/pic/human.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="TwinklingLabel" name="gasSensorLabel">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/pic/gas.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TwinklingLabel</class>
   <extends>QLabel</extends>
   <header>TwinklingLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>num7Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num8Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num9Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num4Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num5Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num6Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num1Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num2Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num3Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num0Btn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numPlusBtn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backBtn</sender>
   <signal>clicked()</signal>
   <receiver>WirelessAlarmWidget</receiver>
   <slot>onSoftKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSoftKeyClicked()</slot>
 </slots>
</ui>
